### PR TITLE
Add docs for API changes for regulations

### PR DESCRIPTION
**Pull request by narsaynorath**
* include parameter for regulation_sections now displays regulations
  with description and category

Refs #SDE-4354

TODO:
- [x] Set milestone label in pull request (SDE version)
- [x] Updated changelog.md
- [x] Completed documentation for the new endpoint.
- [x] Feature has been merged into SDE develop.

### CODE REVIEW

File: source/includes/changelog.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v4.8
+- Included category  and description to regulation_sections include filter for Tasks and specific Task endpoints
```
Comment (Gee19) [inline source/includes/changelog.md]: extra space

File: source/includes/changelog.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v4.8
+- Added `description` and `regulation_name` sections to `regulation_sections` include filter for Tasks and specific Task endpoints
+- Changed `regulation` to `regulation_id` in `regulation_sections` filter for Tasks and Task endpoints
```
Comment (Gee19) [inline source/includes/changelog.md]: I like the wording above, you should put include filter in this line as well

**Comment by Gee19** [comment]
+1

File: source/includes/tasks.md [hunk 3]
```diff
@@ -168,14 +168,18 @@ Content-Type: application/json
         ],
         "regulation_sections": [
             {
+                "description": "Denial of service (DoS) protection and working in a degraded mode during DoS attacks."
```
Comment (rashidg) [inline source/includes/tasks.md]: Missing comma at the end of this line

File: source/includes/tasks.md [hunk 4]
```diff
@@ -168,14 +168,18 @@ Content-Type: application/json
...
                 "id": "RS1026",
                 "name": "CR2.7 (L3 and higher)",
-                "regulation": "CR45"
+                "regulation_id": "CR45"
```
Comment (rashidg) [inline source/includes/tasks.md]: Missing comma at the end

File: source/includes/tasks.md [hunk 5]
```diff
@@ -168,14 +168,18 @@ Content-Type: application/json
...
+                "regulation_name": "ANSI/ISA 62443-4-2",
```
Comment (rashidg) [inline source/includes/tasks.md]: Unnecessary comma

File: source/includes/tasks.md [hunk 6]
```diff
@@ -168,14 +168,18 @@ Content-Type: application/json
...
             },
             {
+                "description": "Denial of service (DoS) protection and working in a degraded mode during DoS attacks."
                 "id": "RS642",
                 "name": "SR2.7 (L3 and higher)",
-                "regulation": "CR39"
+                "regulation_id": "CR39"
+                "regulation_name": "ANSI/ISA 62443-3-3",
```
Comment (rashidg) [inline source/includes/tasks.md]: same 3 comments above

Comment (narsaynorath) [inline source/includes/tasks.md]: Thanks for pointing those out! They're fixed now.

**Comment by rashidg** [comment]
+1

**Pull request merged**